test/config_file.c

```diff
@@ -1092,6 +1100,34 @@ check-purpose=codesign\n";
 	g_assert_true(config->keyring_allow_partial_chain);
 	g_assert_true(config->keyring_check_crl);
 	g_assert_cmpstr(config->keyring_check_purpose, ==, "codesign-rauc");
+
+	g_free(pathname);
```

Comment:
This does not really test the added functions but manually re-implements parts of their internal behavior. Maybe simply assigning allocated strings to each member of ConfigFileOverride is easier in any way?

Reply:
I'd be fine with that.